### DISCUSSION TITLE
Network not managed after latest update

### DISCUSSION THREAD

**Discussion by arnithehope** [elementary/triage#658] [Q&A] [answered]
My system does not manage the network after the latest Upgrade of elementary OS 8.1
I don't know how to reinstall the management software and I can't even start a live system from a stick although usb works aright.
May be it only Ha's to be switched on but I don't know how.
Can somebody whelp me?

**Comment by se290763** [comment]
I have the same problem

**Comment by teamcons** [comment]
starting a grub entry with an older kernel doesnt work?
or apt update, even without network, maybe some script didnt trigger?
or apt --reinstall some driver, the package should be still sitting in the cache, same possible reason of script not running

this sound like some ubuntu mess ._.

**Comment by teamcons** [comment, marked as answer]
From Discord:
 it was nmcli not managing my device for some reason. Solved by creating a blank file in /etc/NetworkManager/conf.d/10-globally-managed-devices.conf

terminal:
> sudo touch /etc/NetworkManager/conf.d/10-globally-managed-devices.conf

**Reply by arnithehope** [reply]
Unbelievable This worked for me, too. Thank you so much. You are the best 💚

**Reply by teamcons** [reply]
:) just sharing the solution another user found
The issue is known, and being studied

Ive had that issue some years ago. It is quite distressing. Glad theres a solution

**Comment by arnithehope** [comment]
Thank you very much. It was the first time I tried to find help in this community. And it's great that it actually worked out.So thank you again and thanks for sharing.Von meinem iPhone gesendetAm 20.12.2024 um 03:20 schrieb Stella and Charlie ***@***.***>:﻿
:) just sharing the solution another user found
The issue is known, and being studied
Ive had that issue some years ago. It is quite distressing. Glad theres a solution

**Comment by se290763** [comment]
It also solved my problem with the network adapter. After a reboot t was back online. Great and simple solution. Thak you  from me too.

**Comment by ryonakano** [comment]
Leaving a note that this issue is fixed in https://github.com/elementary/os/pull/745. Starting from the next daily and stable ISOs this fix takes effect.

**Reply by NateBarham** [reply]
I’m having a similar issue with an install I did today. Networking was fine on initial, devices completely disappear after updates and first restart. Not even visible as unmanaged.

Empty file solution does nothing.

**Reply by ryonakano** [reply]
Did you restart the system after the empty file solution? If yes, yours sounds like a different issue.

**Reply by NateBarham** [reply]
Thanks for the reply!

Yes, this is after restart. I did have the unmanaged error a few days back, and the problem was fixed by changing netplan (no luck there this time, either). That’s why I thought the issues were probably related.

**Reply by ryonakano** [reply]
So you came across the unmanaged error, changed from NetworkManager to netplan and networking worked, but now networking isn't working now again, right?

What about switching back to NetworkManager and then do the empty file solution + restart?

**Reply by NateBarham** [reply]
Sorry. I should have been more clear.

A few days ago (with Elementary iso usb from that day):
unmanaged after updates
netplan solve worked by changing the yaml to include enp8s0

For reasons, uninstalled Elementary

Today (with new iso downloaded today):
no network devices visible in network settings, at all, after updates
empty file solve does not fix
netplan yaml solve does not fix

**Reply by ryonakano** [reply]
Thank you for your clarification. I tried installing from the iso downloaded today, updating, and then checks if the empty file solution works, and it works for me. So your issue might be another thing.

If you execute `lspci` command in the terminal and no line contains `Ethernet controller`, your NIC may not be detected by the kernel.

**Reply by NateBarham** [reply]
Thanks for following up again.

lspci does show lines for both “Network controller” and “Ethernet controller”

Network controller: Realtek Semiconductor Co., Ltd. RTL8852CE PCIe 802.11ax Wireless Network Controller (rev 01)

Ethernet Controller: Realtek Semiconductor Co., Ltd. RTL8125 2.5GbE Controller (rev 05)

Not sure if this info helps, but I thought I’d include it. Also, the Ethernet and WiFi connection works fine until Elementary installs its first automatic round of updates.

**Reply by ryonakano** [reply]
Thank you for the information, so your NIC should be detected by the kernel……I'm sorry but I'm not sure how I can resolve your issue.

> Ethernet Controller: Realtek Semiconductor Co., Ltd. RTL8125 2.5GbE Controller (rev 05)

I have a PC with this controller by chance so I tried installing elementary OS 8 but didn't face your issue.

We're going to release a new OS 8 iso in the near feature (it's in the RC state and available for sponsors at the moment), which includes a fix for Ethernet networks becoming unmanaged after updates, so reinstalling your system using this iso when it's released might resolve your issue.

**Reply by NateBarham** [reply]
That is great news! I appreciate the info. 🙂

Is there a way for me to know when that update occurs? A notification or alert I can set somewhere?

**Reply by ryonakano** [reply]
Note that I'm suggesting reinstallation, not updating your current installation, and plus I'm not sure if that really fixes your issue. I'm sorry if that becomes no avail.

> Is there a way for me to know when that update occurs? A notification or alert I can set somewhere?

I'm not sure if it's officially announced on [our Mastodon](https://mastodon.social/@elementary) or [our blog](https://blog.elementary.io/), but I'm a developer contributing to elementary OS so I can let you know here when the new iso is available, so that you can get to know through the web notification or email notification provided by GitHub.

**Reply by NateBarham** [reply]
Yep. I understand that this would mean a reinstall.

And it is very generous of you to provide the reminder. I look forward to it!

**Reply by ryonakano** [reply]
@NateBarham Letting you know that elementary OS 8.0.1 is now available from [our website](https://elementary.io/ja/), which includes fixes for the original issue discussed in this discussion. You can reinstall and check whether your issue is fixed too if you're still using elementary OS.

**Comment by NateBarham** [comment]
Thanks for letting me know! I’ll give it a look when I get the chance. Still hoping to have an Elementary partition going alongside Windows.

**Reply by NateBarham** [reply]
Sadly, had the chance to install the new build of Elementary. And just as before, networking is fine, downloads work for software updates and web. Then after first restart all networking options disappear. 😔

**Reply by ryonakano** [reply]
Hmm, then your issue seems to be different from the issue originally discussed in this Discussion.
